src/tools/pontoon/resources.md

```diff
@@ -4,30 +4,35 @@
 
 Pontoon features built-in access to different translation memories. These resources are available in the translation workspace and, for machinery, on a [standalone web page](https://pontoon.mozilla.org/machinery/).
 
-## History tab
+## History
 
-The history tab shows all of the suggestions and translations that have been submitted for the current source string.
-
-In the tab, the number of available entries is visible alongside the `HISTORY` title. The history entries are located below the tab.
+Pontoon displays the translation history for strings below the editor. The current approved translation for a source string appears at the top and all previous suggestions and translations beneath.
 
 ![History tab](../../assets/images/pontoon/resources/history.png)
 
-On the left side, above the entry, the name of the translator is visible in gray (linking to their profile), while in green it’s displayed how long ago the entry was submitted (full date and time is available as a tooltip). Below this information, the translation or suggestion is displayed.
+Each translation history entry contains:
+* The name of the translator and their profile picture (linking to their profile).
+* How long ago the entry was submitted (hover over to see the full date and time is available as a tooltip).
+* The translation or suggestion.
+* Icons indicating translation status (see below).
+* Translation comments. You can toggle `COMMENT` to show all comments for a translation or to add a new comment.
```

Comment:
I think we can differentiate this. There will be the number of comments displayed, if there are any.